ptftplib/tftpclient.py

```diff
@@ -565,13 +565,13 @@ def get(self, args):
                   .format(filename))
             return False
 
+        transfer_speed = self.__get_speed(self.PTFTP_STATE.filesize, transfer_time)
+
         print('Transfer complete, {} bytes ({:.2f} kB/s)'
-              .format(self.PTFTP_STATE.filesize,
-                      self.__get_speed(self.PTFTP_STATE.filesize,
-                                       transfer_time)))
+              .format(self.PTFTP_STATE.filesize, transfer_speed))
         self.PTFTP_STATE.file.close()
```

Comment:
While you're in there, would you mind moving this and the line below (the close and the remove) in a `finally` block of the try/except above (line 567)?